spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/support/jackson/JobParametersJacksonMixIn.java

```diff
@@ -27,7 +27,7 @@
  * @author Gunnar Hillert
  * @since 1.0
  */
-@JsonIgnoreProperties("empty")
+@JsonIgnoreProperties({"empty", "identifyingParameters"})
```

Comment:
Why do we want to ignore `identifyingParameters`?

Reply:
This attribute returns an unmodifiableMap and thus when it deseriallizes it fails.  This change is a shim awaiting the merge of https://github.com/spring-cloud/spring-cloud-dataflow/pull/5850.